src/security/cli_security.c

```diff
@@ -177,11 +178,14 @@ sanity_check_credential_response(Drpc__Response *response)
 			response->body.len, response->body.data);
 	if (pb_cred == NULL) {
 		/* Malformed body */
+		D_ERROR("Unable to unmarshal credential: body.len: %zu\n",
+			response->body.len);
 		return -DER_MISC;
 	}
 
 	/* Not super useful if we didn't get a token */
 	if (pb_cred->token == NULL) {
+		D_ERROR("Credential has no token\n");
```

Comment:
Could you please elaborate on when this case can happen?

Reply:
I don't think there is a situation where this would happen. This is more of a sanity check against a bug in the software.